src/racing/actors.c

```diff
@@ -2517,121 +1426,121 @@ s16 addActorToEmptySlot(Vec3f pos, Vec3s rot, Vec3f velocity, s16 actorType) {
     return -1;
 }
 
-s16 func_8029ED38(Vec3f pos, s16 actorType) {
+s16 allocate_actor_with_new_position(Vec3f pos, s16 actorType) {
```

Comment:
Can you add UNUSED keyword if this func is unused? (It appears to be).

`spawn_actor_at_pos`